src/app/projects/project-portfolio-wizard/project-portfolio-wizard.tpl.html

```diff
@@ -1,5 +1,5 @@
-<div>
```

Comment:
We would like all directives to be in a root `div` element with its class name. This is because `replace: true` is on some, but not all, of the directives. Some were not updated during the refactor. Can you add this back in and add the class `project-portfolio-wizard`?